src/main.cpp

```diff
@@ -413,11 +415,15 @@ int main(int argc, char** argv) {
                   << "sec\n";
     }
 
-    if (Global::config().get("provenance") == "1"){ 
+    if (Global::config().has("provenance")) {
         // construct SouffleProgram from env
         SouffleInterpreterInterface interface(*env, translationUnit->getSymbolTable());
         // invoke explain
-        explain(interface);
+        if (Global::config().get("provenance") == "1") {
```

Comment:
Instead of '1' and '2' could you make this more intuitive? Something like 'interactive' instead of '1' and  'non-interactive' instead of '2'